include/role/glob.h

```diff
@@ -24,5 +24,7 @@
 
 #define LIBROLE_ROLE_EXTENSION ".role"
 
+#define LIBROLE_MAX_SYSTEM_ROLES 256
```

Comment:
Why such a small limit?

Reply:
I don't think is so small. It is a roles on a local machine. It can be user, localadmin, poweruser, ... Ok, How much we need?